services/web/client/source/class/qxapp/data/Store.js

```diff
@@ -4,32 +4,53 @@ qx.Class.define("qxapp.data.Store", {
   type : "singleton",
 
   events: {
-    "servicesRegistered": "qx.event.type.Event"
+    "servicesRegistered": "qx.event.type.Event",
+    "interactiveServicesRegistered": "qx.event.type.Event"
   },
 
   members: {
-    getServices: function() {
+    getComputationalServices: function() {
       let req = new qx.io.request.Xhr();
       req.set({
-        url: "/repositories",
+        url: "/get_computational_services",
         method: "GET"
       });
       req.addListener("success", function(e) {
         let requ = e.getTarget();
         const listOfRepositories = JSON.parse(requ.getResponse());
+        console.log("listOfServices", listOfRepositories);
         let services = [];
         for (const key of Object.keys(listOfRepositories)) {
           const repo = listOfRepositories[key];
           const nTags = repo.length;
           for (let i=0; i<nTags; i++) {
-            console.log(i, repo[i]);
             let newMetadata = qxapp.data.Converters.registryToMetadata(repo[i]);
             services.push(newMetadata);
           }
         }
         this.fireDataEvent("servicesRegistered", services);
       }, this);
       req.send();
+    },
+
+    getInteractiveServices: function() {
+      let socket = qxapp.wrappers.WebSocket.getInstance();
+      socket.removeSlot("getInteractiveServices");
```

Comment:
I would use REST for calls like this where there is no bidirectionality required ...

Reply:
Agree, at some point next sprint, we will standarize the communication channels